packages/tezos-signer-backends/tezos-signer-backends.15.1/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+maintainer: "contact@tezos.com"
+authors: ["Tezos devteam"]
+homepage: "https://www.tezos.com/"
+bug-reports: "https://gitlab.com/tezos/tezos/issues"
+dev-repo: "git+https://gitlab.com/tezos/tezos.git"
+license: "MIT"
+depends: [
+  "dune" { >= "3.0" }
+  "tezos-base" { = version }
+  "tezos-stdlib" { = version }
+  "tezos-client-base" { = version }
+  "tezos-rpc-http" { = version }
+  "tezos-rpc-http-client" { = version }
+  "tezos-signer-services" { = version }
+  "tezos-shell-services" { = version }
+  "uri" { >= "2.2.0" }
+  "tezos-error-monad" { with-test & = version }
+  "tezos-crypto" { with-test & = version }
+  "alcotest-lwt" { with-test & >= "1.5.0" }
+  "ocplib-endian"
+  "fmt" { >= "0.8.7" }
+  "tezos-clic" { = version }
+  "tezos-stdlib-unix" { = version }
+  "tezos-rpc-http-client-unix" { = version }
+]
+depopts: [
+  "ledgerwallet-tezos"
+]
+conflicts: [
+  "ledgerwallet-tezos" { < "0.2.1" }
```

Comment:
```suggestion
  "ledgerwallet-tezos" { < "0.2.1" | >= "0.3.0" }
```

There was a breaking-change in the API going into the 0.3.x versions. This is causing the CI to fail.